sc2/bot_ai.py

```diff
@@ -698,6 +734,7 @@ def _prepare_step(self, state):
             self._previous_upgrades = self.state.upgrades
 
     async def _issue_unit_added_events(self):
+        """ Its called when an unit is created on when a building is started, separating both"""
```

Comment:
Aside from grammar (Its/It's, an/a) this comment isn't really accurate. Since it's an internal function I don't think it needs documentation, but if you want something, how about
```suggestion
        """ Calls self.on_unit_created for each new unit and self.on_building_started for each started building. """
```

Reply:
You right, but even tho it's internal I would like to document it for consistency